admin/lib/generators/solidus_admin/component/component_generator.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  class ComponentGenerator < Rails::Generators::NamedBase
+    source_root File.expand_path('templates', __dir__)
+
+    argument :attributes, type: :array, default: [], banner: "attribute"
+
+    class_option :html, type: :boolean, default: true
+    class_option :i18n, type: :boolean, default: true
```

Comment:
I'd like to enforce the good practice of always using i18n, especially if it's so easy to get it modular for the component. Do you think it's common for people to not use i18n, even if it's so straightforward?

Reply:
Ah right, I think it's hard to anticipate that you won't need any text before building the component but could happen and as long as the default is `true`, I think it's ok to keep the option.